contributor/04-mechanics-of-contributing.md

```diff
@@ -0,0 +1,125 @@
+
+## Outline doc
+
+See [Outline Section](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/outline.md#mechanics-of-contributing)
+
+# Article
+
+You are ready to start contributing to other teams projects/repos or look forward to reduce your blockers by not playing the big-cheese game but by collaborating? 
+Please consider reading this section beforehand as it will give you practical advice and informs you about gotchas to remember when making an InnerSource contribution. This will hopefully enable you and the host team to have as pleasant an experience as possible setting the foundation for more contributions and great collaboration.  
+
+This article is separated into the three steps you will likely experience
+
+- Soliciting your contribution opportunity and preparing to work on it
+- Actually creating the contribution
+- Polishing and wrapping the gift nicely and presenting it to the host team.
+
+If your contribution is larger, you'll possibly go through (some) these steps repeatedly as you iterate towards your common goal.
+It is very likely that as you do this everything will feel more and more naturally - maybe you'll even wonder why you were doing anything else before.
+
+## Preparing to work
+
+### Lead times
+
+One key difference is the turn-around time. 
+You're coming to a new (host) team on every first contribution and thus you'll need to get to know their approach, i.e. their development, reviews and deployment, etc. processes.
+Even if you are coming back after a while the teams' ways and members might have changed - take your time as you would to catch up with a friend you haven't seen in a while and whom you are visiting now.
+
+Thus: Please give yourself enough lead time.
+Start early enough so that your work is available for you to leverage at the time you need it. 
+It's better to add more slack time initially - you'll get a feeling about the turn-around times once you work with the host team.
+If you end with too much there surely will be a great way to make use of the time, be it some work on your side, the quirky bug or improvement opportunity you found while making your initial contribution or possibly even a nice chat over coffee, beers in case of personal availability or whatever topic in case of distributed projects. 
+
+[EDITOR NOTE: There was actually a paper on how trust is build up on remote/asynchronous collaboration. Maybe I'll find that again. 
+The basic point is that aside of reliability in collaboration, continued presence the effect of water cooler discussions and food/drink consumption is quite impressive...
+]  
+
+Especially in the case of larger distances between you and the host teams, e.g. multiple time zones difference as in San Francisco, USA (PDT) and Berlin, Germany (CEST), you will need to actively consider this and explicitly manage your and their expectations on reaction times.
+
```

Comment:
### Building trust

InnerSource puts huge weight on written communication - in particular when it comes to project decisions. Does that imply that in-person communication is forbidden?

Clearly not: Where written communication shines when it comes to archiving and searchability, in-person communication shines when it comes to communication bandwidth. Try to make time to meet the people behind the names. If possible, try to meet them over your favorite beverage or some food. When you're able to hear people speak, when you know their idiosyncrasies remote collaboration will become easier. 

[EDITOR NOTE: There was actually a paper on how trust is build up on remote/asynchronous collaboration. Maybe I'll find that again. 
The basic point is that aside of reliability in collaboration, continued presence the effect of water cooler discussions and food/drink consumption is quite impressive...
]